pyqt-apps/requirements.txt

```diff
@@ -1,2 +1,2 @@
 pydm==1.8.0
-QtAwesome==0.6.1
```

Comment:
I think it is a good idea that we use loose version conditions like
`QtAwesome>=0.6.1` in the requerements file and let ansible control precisely package versions.
This version should be updates in Ansible as well.

Reply:
Done!